Xkcd-ComicsTests/Xkcd_ComicsTests.swift

```diff
@@ -25,7 +25,10 @@ class XkcdComicsTests: XCTestCase {
         // Given
         let comic = Comic(num: 1, title: "Test", img: "Test", alt: "Test", year: "Test", month: "Test")
         // When
-        let viewModel = ComicsVM()
+        let viewModel = ComicsViewModel(
+            fetchingService: <#FetchingService#>,
```

Comment:
Placeholders in code

Reply:
I plan to create mockData in a new branch